src/CSI.IBTA.Administrator/Clients/UserServiceClient.cs

```diff
@@ -51,5 +52,38 @@ public async Task<GenericInternalResponse<UserDto>> GetUser(int userId)
 
             return new GenericInternalResponse<UserDto>(false, null, user);
         }
+
+        public async Task<IQueryable<SettingsDto>?> GetEmployerSettings(int employerId)
+        {
+            var response = await _httpClient.GetAsync(UserServiceApiEndpoints.Settings + "/" + employerId);
+
+            if (!response.IsSuccessStatusCode)
+            {
+                _logger.LogError("Request unsuccessful");
+                return null;
+            }
+
+            var responseContent = await response.Content.ReadAsStringAsync();
+            var employersSettings = JsonConvert.DeserializeObject<List<SettingsDto>>(responseContent).AsQueryable();
+
+            return employersSettings;
+        }
+
+        public async Task<IQueryable<SettingsDto>?> UpdateEmployerSettings(int employerId, List<SettingsDto>? SettingsDtos)
+        {
+            var content = JsonContent.Create(SettingsDtos);
+            var response = await _httpClient.PatchAsync(UserServiceApiEndpoints.Settings + "/" + employerId, content);
```

Comment:
It is better to use string interpolation. https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated

When you concatenate strings with a + sign, you end up creating multiple strings to get to the final string.

When you use the interpolation method (or StringBuilder), the .NET runtime optimizes your string use, so it (in theory) has better memory usage.